content/swiftui/concepts/views/terms/roundedRectangle/roundedRectangle.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Rounded Rectangle'
+Description: 'A rectangular shape with rounded corners inside the frame of the view containing it.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Rounded Rectangle'
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`RoundedRectangle`** view is a pre-built rectangular shape with rounded corners that is aligned inside the frame of the `View` containing it, and can be visually modified as desired.
+
+SwiftUI has several built-in shapes, such as a rectangle, ellipse and capsule.
```

Comment:
Add links to the other shape entries.

Reply:
I don't know how to link shapes that are in a PR. can you help me out please?

Reply:
I am trying to change it but results in an error. Any idea why?
<img width="564" alt="Screenshot 2022-12-02 at 10 51 23" src="https://user-images.githubusercontent.com/45493612/205276614-7277a48a-d0ca-494c-852a-a3e3d921d231.png">

Reply:
Perfect, thank you!!